app/api/proxy/urls.py

```diff
@@ -30,6 +30,11 @@
         ProxyView.as_view(upstream=f"{settings.NEXTJS_URL}/scanreports/"),
         name="scan-report-edit-table",
     ),
+    re_path(
```

Comment:
As this view isn't complete, and we depend on the modals being finished before it is releasable, could this url path be configured by an environment variable? Just like `ENABLE_PROXY` in the config urls, and default to off.